apps/antalmanac/src/components/RightPane/CoursePane/SearchForm/AdvancedSearch/AdvancedSearchTextFields.tsx

```
@@ -60,7 +54,7 @@ export function AdvancedSearchTextFields() {
             const urlParam = new URLSearchParams(url.search);
```

I know you didn't write this function, but, while you're here, could you refactor it a bit?

1) This function is not a change handler. It's a change handler factory. (I.e., it's a function that makes change handlers.) Therefore, it should be called `changeHandlerFactory`.
2) The default case in the switch statement is a silent failure. It should at least `console.error`.

CC: @KevinWu098 because the git blame says you wrote it.